andela/ah-commando-backend
# 167190423 Configure Eslint

####  What does this PR do?
Configures eslint, prettier and pre-commit hook (husky)
#### Description of Task to be completed?
* created .eslintrc file
* created .prettierrc file
* added husky hook to enforce eslint
#### How should this be manually tested?
* pull or fetch this branch
* run npm install to add necessary files
* install eslint and prettier extension for vscode or your editor
* for vscode, insert "editor.formatOnSave": true in users settings
* create a .js file and paste/write a code to see eslint and prettier in action
#### Any background context you want to provide?
No
#### What are the relevant pivotal tracker stories?
[#167190423](https://www.pivotaltracker.com/story/show/167190423)
#### Screenshots (if appropriate)
<img width="651" alt="Screenshot 2019-07-22 at 3 57 49 PM" src="https://user-images.githubusercontent.com/37886814/61643424-c1549e00-ac9a-11e9-9aab-b027d4a9a014.png">
<img width="706" alt="Screenshot 2019-07-22 at 3 58 51 PM" src="https://user-images.githubusercontent.com/37886814/61643437-c7e31580-ac9a-11e9-8d4d-e66a0ffab52e.png">

#### Questions:
No
